R/plotPercentiles.R

```diff
@@ -0,0 +1,108 @@
+#' Comparison plots show 50th percentile of user selected variables as obtained from different scenario runs. If
+#' available in the data, ribbon plots will also show the 33th - 67th percentile in a darker color and the 5th – 95th
+#' percentile in a lighter color. *Note*: the 5th, 33th, 67th and 95th percentile must be given in the provided data set
+#' as the percentiles are not computed
+
+#' @author Tonn Rueter
+#' @param df `quitte` style data frame containing all variables for each scenario. In the quitte data frame all
+#'    percentiles must be given as individual variables. Manipulate input data frame such that all percentiles
```

Comment:
" In the quitte data frame all  percentiles must be given as individual variables ..."
Please add an example to avoid misunderstandings.

"Manipulate input data frame ....." 
This should not be here, as it is code documentation, but not a specification of the input parameter and therefore nothing a user has to worry about, right?